service/hook/gitlab/gitlab.go

```diff
@@ -160,6 +169,13 @@ func (branchInfoModel BranchInfoModel) getRepositoryURL() string {
 	return branchInfoModel.GitSSHURL
 }
 
+func (repository RepositoryModel) getRepositoryURL() string {
+	if repository.VisibilityLevel == 20 {
```

Comment:
It would help the reader if you moved this to a well-named constant. I can guess it's the public level because of the next line, but it was a bit confusing at first.